libs/mep/ace0861/section-metadata/section-metadata.css

```
@@ -0,0 +1,398 @@
+.section-metadata {
+  display: none;
+}
+
+.section.darkest {
+  background-color: rgb(0 0 0);
+  color: rgb(255 255 255);
+}
+
+.section.dark {
+  background-color: rgb(29 29 29);
+  color: rgb(255 255 255);
+}
+
+.section.light {
+  background-color: rgb(255 255 255);
+  color: rgb(34 34 34);
+}
+
+.section.hide-sticky-section,
+.section.close-sticky-section {
+  display: none;
+}
+
+.section.has-background {
+  background-color: unset;
+}
+
+.section.xxxl-spacing-static {
+  padding: var(--spacing-xxxl-static) 0;
+}
+
+.section.xxl-spacing-static {
+  padding: var(--spacing-xxl-static) 0;
+}
+
+.section.xl-spacing-static {
+  padding: var(--spacing-xl-static) 0;
+}
+
+.section.xxxl-spacing {
+  padding: var(--spacing-xxxl) 0;
+}
+
+.section.xxl-spacing {
+  padding: var(--spacing-xxl) 0;
+}
+
+.section.xl-spacing {
+  padding: var(--spacing-xl) 0;
+}
+
+.section.l-spacing {
+  padding: var(--spacing-l) 0;
+}
+
+.section.m-spacing {
+  padding: var(--spacing-m) 0;
+}
+
+.section.s-spacing {
+  padding: var(--spacing-s) 0;
+}
+
+.section.xs-spacing {
+  padding: var(--spacing-xs) 0;
+}
+
+.section.xxs-spacing {
+  padding: var(--spacing-xxs) 0;
+}
+
+.section.xxxl-padding {
+  padding: var(--spacing-xxxl);
+}
+
+.section.xxl-padding {
+  padding: var(--spacing-xxl);
+}
+
+.section.xl-padding {
+  padding: var(--spacing-xl);
+}
+
+.section.l-padding {
+  padding: var(--spacing-l);
+}
+
+.section.m-padding {
+  padding: var(--spacing-m);
+}
+
+.section.s-padding {
+  padding: var(--spacing-s);
+}
+
+.section.xs-padding {
+  padding: var(--spacing-xs);
+}
+
+.section.xxs-padding {
+  padding: var(--spacing-xxs);
+}
+
+.section div.section-background {
+  display: block;
+  position: absolute;
+  inset: 0;
+  z-index: -1;
+  height: 100%;
+  width: 100%;
+}
+
+.section picture.section-background {
+  display: block;
+  position: absolute;
+  inset: 0;
+  z-index: -1;
+}
```

```suggestion
.section .section-background {
  display: block;
  position: absolute;
  inset: 0;
  z-index: -1;
}

.section div.section-background {
  height: 100%;
  width: 100%;
}
```
Note for RTP: QA doing this instead.  But this may need more extensive QA since it's not quarantined to a new variant.